_meta/config/common.p2.yml.tmpl

```
@@ -32,6 +33,18 @@ inputs:
         - filesystem
         data_stream.dataset: system.filesystem
 
+  # Collecting log files
+  #- type: filestream
+  #  id: your-input-id:
```

```suggestion
  #  id: your-input-id
```

There's an unnecessary colon.